File: AEDS/EXERCÍCIOS/NIVELAMENTO2/exercicio1.java
import java.util.Scanner; import java.io.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String nomeArq = sc.nextLine();
        String frase = sc.nextLine();
        try (FileWriter arq = new FileWriter(nomeArq)) {
            arq.write(frase);
        } catch (IOException e) {
            System.out.println(e.getMessage());
        }
        try (BufferedReader br = new BufferedReader(new FileReader(nomeArq))) {
            String fraseLida = br.readLine();
            System.out.println(fraseLida);
        } catch (IOException e) {
            System.out.println(e.getMessage());
        }
        try (BufferedReader br = new BufferedReader(new FileReader(nomeArq))) {
            String fraseLida = br.readLine();
            String fraseMaiuscula = fraseLida.toUpperCase();
            System.out.println(fraseMaiuscula);
        } catch (IOException e) {
            System.out.println(e.getMessage());
        }
        String nomeArq2 = sc.nextLine();
        try (BufferedReader br = new BufferedReader(new FileReader(nomeArq));
             BufferedWriter bw = new BufferedWriter(new FileWriter(nomeArq2))) {
            String fraseLida = br.readLine();
            String fraseMaiuscula = fraseLida.toUpperCase();
            bw.write(fraseMaiuscula);
        } catch (IOException e) {
            System.out.println(e.getMessage());
        }
        sc.close();
    }
}

File: AEDS/EXERCÍCIOS/NIVELAMENTO2/exercicio1.c
#include <stdlib.h>
#include <stdio.h>
#include <ctype.h>

int main () {
    char nomeArq[100]; scanf("%s", nomeArq);
    FILE *arq = fopen(nomeArq, "w");
    char frase[100]; scanf(" %[^\n]", frase);
    fprintf(arq, "%s", frase);
    fclose(arq);
    arq = fopen(nomeArq, "r");
    char fraseLida[100];
    fscanf(arq, " %[^\n]", fraseLida);
    printf("%s\n", fraseLida);
    fclose(arq);
    for (int i = 0; fraseLida[i] != '\0'; i++) fraseLida[i] = toupper(fraseLida[i]);
    printf("%s\n", fraseLida);
    char nomeArq2[100]; scanf("%s", nomeArq2);
    FILE *arq2 = fopen(nomeArq2, "w");
    fprintf(arq2, "%s", fraseLida);
    fclose(arq2);
}
